plugins/woocommerce/README.md

```diff
@@ -64,7 +64,7 @@ pnpm --filter='@woocommerce/admin-library' test:watch
 pnpm --filter='@woocommerce/admin-library' ts:check
 ```
 
-### `plugins/woocommerce-blocks`
+### `plugins/woocommerce/client/blocks`
```

Comment:
Is this section necessary anymore? It mentions commands to specifically build blocks but doesn't `  pnpm --filter='@woocommerce/plugin-woocommerce' build` build blocks anyway? Feels like we're adding confusion.

Reply:
These might still be relevant when running an individual Blocks build. It's also consistent with the Admin readme above, so if it requires any update, we should do it in a separate PR. Does that make sense?